h2o-docs/src/product/data-science/glrm.rst

```diff
@@ -125,6 +152,10 @@ Below is a simple example showing how to build a Generalized Low Rank model.
     # Generate predictions on a validation set (if necessary):
     arrests_pred <- h2o.predict(glrm_model, newdata = valid)
 
+    # Transform the data using the validation set to retrieve the new coefficients:
+    glrm_transform <- h2o.transform_frame(glrm_model, valid)
```

Comment:
@hannah-tillman Can you change the wording:

using the validation set

to 

using the dataset valid